.github/workflows/ci.yml

```diff
@@ -134,10 +134,10 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2.3.4
-    - name: Setup Python 3.8
+    - name: Setup Python 3.9
       uses: actions/setup-python@v2.2.2
       with:
-        python-version: 3.8
+        python-version: 3.9
```

Comment:
I noticed it is possible to do `'3.x'`, which would save needing to keep updating this one.

Reply:
FWIW 3.x is probably the default so the whole `with:` could be dropped. By the way, the whole Composite GitHub Actions thing got a bit better and almost usable recently so maybe we'll be able to even deduplicate the maintenance efforts (I've been experimenting with this and other CI improvements over the past year, hope to apply some in aio-libs as well).

Reply:
Done, thanks for the suggestions!